pkg/crc/config/settings.go

```diff
@@ -31,6 +31,7 @@ const (
 	KubeAdminPassword       = "kubeadmin-password"
 	Preset                  = "preset"
 	EnableSharedDirs        = "enable-shared-dirs"
+	SharedDirPassword       = "shared-dir-password" // #nosec G101
```

Comment:
@gbraad Changed this to `shared-dir-password` instead of `smb-share-password` more generic, can be reused when we do the sshfs

Reply:
password aren't shared among dirs, as they apply to each individual share.

Reply:
ahh, but i couldn't find a way to set password per share in windows, either from the GUI or from the the smbshare powerhsell cmdlets

Reply:
Windows bases that on the UNC path; they can be assigned a different user and password combination. In the current case of CIFS use you only deal with a USERFOLDER and not 'arbitrary' options given by the user.